Pipfile

```diff
@@ -27,7 +27,8 @@ autopep8 = "*"
 # Pins to make lockfile usable on multiple Python versions and platforms
 typing-extensions = "*"
 # All other package requirements from setup.cfg
-ibek = {editable = true, path = "."}
+ibek = {editable = true,path = "."}
+jsonschema = "*"
```

Comment:
This should go in setup.cfg

Reply:
Thanks this is an easy thing to forget and we dont have checks for it on github. We do have dls-py3 internally. Should we do something about this?

Reply:
I'm happy to just rely on gitlab CI for this, rather than make github do it too